src/main/java/by/lawaksoft/tradebot/config/OkxApiConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package by.lawaksoft.tradebot.config;
+
+import by.lawaksoft.tradebot.dto.config.APIConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableFeignClients("by.lawaksoft.tradebot.client")
+@ComponentScan("by.lawaksoft.tradebot")
```

Comment:
I suppose, it's better move to AppConfig class (you can create new, if we haven't this now)
 Also EnableFeignClients is better to move into AppConfig 
 
 This annotations (ComponentScan, EnableFeignClients) belong to application, not to OkxApi

Reply:
done